grails-app/utils/org/pih/warehouse/OutboundWorkflowState.groovy

```diff
@@ -0,0 +1,31 @@
+package org.pih.warehouse
```

Comment:
This should just go with all of the other enum classes under src/main/groovy.

The package can just be org.pih.warehouse.api for now. But this is very UI specific so we might need to come up with another convention for that. 

Note: I made a conscious decision to call this State rather than Status because it represents arbitrary states of the UI workflow. We should never have tied these states (or steps) to the API - that was my fault. Instead each UI workflow step should have been working against a separate resource related to the stock movement.